README.md

```diff
@@ -145,12 +145,14 @@ The following table lists the configurable parameters of the Harbor chart and th
 | `logLevel`                                                                  | The log level                                                                                                                                                                                                                                                                                                                                   | `debug`                         |
 | `harborAdminPassword`                                                       | The initial password of Harbor admin. Change it from portal after launching Harbor                                                                                                                                                                                                                                                              | `Harbor12345`                   |
 | `secretkey`                                                                 | The key used for encryption. Must be a string of 16 chars                                                                                                                                                                                                                                                                                       | `not-a-secure-key`              |
 | **Nginx** (if expose the service via `ingress`, the Nginx will not be used) |
 | `nginx.image.repository`                                                    | Image repository                                                                                                                                                                                                                                                                                                                                | `goharbor/nginx-photon`         |
 | `nginx.image.tag`                                                           | Image tag                                                                                                                                                                                                                                                                                                                                       | `dev`                           |
+| `nginx.image.imagePullSecrets`                                              | Add imagePullSecret names to the nginx deployment                                                                                                                                                                                                                                                                                               |                                 |
```

Comment:
I think these items are not needed anymore either

Reply:
Uhh sorry, I forgot.